webapplication/sip/settings.py

```diff
@@ -251,4 +251,5 @@
 SITE_ID = 1
 
 DEFAULT_SYSTEM_NOTIFICATION_EMAIL = ""
+DEFAULT_REMOTE_SERVER = "http://10.8.0.8:8080"
```

Comment:
Please read this from the settings.GEOAP_CREDS

Reply:
fixed